zebedee-cms/src/main/java/com/github/onsdigital/zebedee/configuration/Configuration.java

```diff
@@ -107,6 +113,10 @@ public static String getBabbageUrl() {
         return StringUtils.defaultIfBlank(getValue("BABBAGE_URL"), DEFAULT_WEBSITE_URL);
     }
 
+    public static String getCacheAPIUrl() {
```

Comment:
`getLegacyCacheAPIUrl`, for consistency.

However, this is not being used anywhere.